CMakeLists.txt

```diff
@@ -13,6 +14,7 @@ if(CMAKE_BUILD_TYPE IN_LIST SIMPLE_TYPES)
 else()
   message(STATUS "CMAKE_BUILD_TYPE = ${CMAKE_BUILD_TYPE}")
 endif()
+message(STATUS "Compiler ${CMAKE_CXX_COMPILER_ID}")
```

Comment:
CMake already reports the compiler for you.

Reply:
I didn't see it before, but now I do. I'll remove this as well.